.changeset/dull-keys-punch.md

```diff
@@ -0,0 +1,6 @@
+---
+'create-amplify': patch
+'@aws-amplify/backend-data': patch
```

Comment:
this should be `minor` given break in api.
(while we're at 0.x minor bump should be used to signal break)